EIPS/eip-2583.md

```diff
@@ -0,0 +1,295 @@
+---
+eip: 2583
+title: Penalty for account trie misses
+author: Martin Holst Swende (@holiman)
+discussions-to: https://ethereum-magicians.org/t/eip-2583-penalties-for-trie-misses/4190
+status: Draft
+type: Standards Track
+category: Core
+created: 2020-02-21
+---
+
+
+## Simple Summary
+
+This EIP propose to introduce a gas penalty for opcodes which access the account for trie non-existent accounts.
```

Comment:
EIPs don't propose, they assert.  One may propose an EIP be adopted, but the EIP itself is an assertion of some change.